demos/common/python/openvino/model_zoo/model_api/adapters/openvino_adapter.py

```diff
@@ -157,3 +160,27 @@ def operations_by_type(self, operation_type):
                 layer_name = node.get_friendly_name()
                 layers_info[layer_name] = Metadata(type=node.get_type_name(), meta=node.get_attributes())
         return layers_info
+
+
+def get_input_shape(input_tensor):
```

Comment:
I can, but I believe it can be adapter specific (different adapters can have different utilities for the same purposes, naming can be misleading), but if it is ok for you for now, then I move

Reply:
I see... Probably the shape will become a separate entity like `Layout`, since different adapters might apply complex processing of shape.
So let's don't move it yet